tests/toranj/README.md

```diff
@@ -12,7 +12,9 @@
 
 ## Setup
 
-`toranj` requires wpantund to be installed. Please follow [`wpantund` installation guide](https://github.com/openthread/wpantund/blob/master/INSTALL.md#wpantund-installation-guide).
+`toranj` requires `wpantund` to be installed.
+- Please follow [`wpantund` installation guide](https://github.com/openthread/wpantund/blob/master/INSTALL.md#wpantund-installation-guide). Note that `toranj` expects `wpantund` installed from latest master branch.
+- Alternative way to install `wpantund` is to use the same commands from travis `before_install` [script](https://github.com/openthread/openthread/blob/master/.travis/before_install.sh#L110) for build target `toranj-test-framework`.
```

Comment:
Prefer to link with specific commit in case the `before_install.sh` is modified.

Reply:
Great idea. Thanks.